hangman.rb

```diff
@@ -15,17 +14,50 @@
 puts prompt
 puts "Your word has ", letter_array.length, "letters"
 puts prompt
-puts blank_array
+puts blank_array.join()
 puts prompt
 puts "You may begin guessing."
 puts prompt
-for x in 1..guesses
+
+  #gues1
+  iftrue = false
+  finish = true
+  game_end = false
+while guesses >= 0 && game_end == false
```

Comment:
So right now when guesses is 0, it'll still run. So the first thing is to switch it to `while guesses > 0`. The second thing is that you can actually do `!game_end` instead of `game_end == false`.

```rb
while guesses > 0 && !game_end
```

Since `game_end` is a boolean, ! means the opposite.